app/components/event-card.js

```diff
@@ -39,4 +39,8 @@ export default class EventCard extends Component {
   selectEventType(eventType) {
     this.set('eventType', eventType === this.eventType ? null : eventType);
   }
+
+  isSingleDay() {
+    return this.startsAt.isSame(this.endsAt, 'day');
```

Comment:
here it should be ```this.event.startsAt``` and ```this.event.endsAt```

Reply:
same result

Reply:
Maybe it is checking `day` as in Friday instead of date